paper-chip-input-autocomplete.html

```diff
@@ -418,6 +418,13 @@
 			_saveTag(name) {
 				if (this.items.indexOf(name) == -1) {
 					this.push('items', name);
+					this.dispatchEvent(new CustomEvent('chip-added', {
```

Comment:
Can you name the CustomEvent name like in the paper-chip? ```chip-created```

Reply:
Fixed :)